stacks/apps/dovetail-counts.yml

```diff
@@ -92,6 +93,13 @@ Resources:
             StartingPosition: LATEST
             Stream: !Ref DovetailCdnLogsKinesisArn
           Type: Kinesis
+        CountsBytesKinesisTrigger:
```

Comment:
Added a 2nd trigger to all these lambdas.  After we deploy and the old input stream stops getting new records - I'll remove the old triggers.

(For this lambda, will require an update to CloudFront to get realtime logs moved to this new stream).